libraries/messages/src/sender.rs

```diff
@@ -9,5 +9,16 @@ use ts_rs::TS;
 #[cfg_attr(feature = "ts", ts(export))]
 pub enum Sender {
     GroundStation,
-    MainBoard,
+    LogicBoard,
+    RecoveryBoard,
 }
+
+impl From<Sender> for u16 {
```

Comment:
You could have the CAN take a Sender for the ID

Reply:
Why? The boards should never change their ID